tests/test_e2e_18_mef_eline.py

```diff
@@ -69,6 +69,7 @@ def assert_tag_used_by_interface(self, interface_id, tag_type, value):
                 break
         else:
             assert True, available_tags
+            return
```

Comment:
This fix looks good, but we didn't end up using the method [`is_tag_used_by_interface`](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/test_e2e_18_mef_eline.py#L38) that you introduced, could you refactor both `assert_tag_used_by_interface` and `assert_tag_not_used_by_interface` to reuse that method, then include the available_tags too be returned too to still have detailed info in the assert messages. 

@Ktmi send a new PR for this fix just so the nightly tests don't keep failing, thanks.